taxi/views.py

```diff
@@ -1,3 +1,18 @@
 from django.shortcuts import render
 
-# Create your views here.
+from .models import Car, Driver, Manufacturer
```

Comment:
Order and group your imports to follow PEP-8 import rules here and in the other files